### PR TITLE
Update Cooler.md

### PR DESCRIPTION
**Pull request by icedterminal**
This needed a total overhaul. Inaccurate info and no links to any tool that may help control one.

I use an NZXT X62 without issue in macOS.

### CODE REVIEW

**Comment by khronokernel** [comment]
Looks good to me, though seems like a bit too much info if I'm being honest. Could've removed most of it, and just mentioned the important info:

- NZXT loose their profile if the system's completely powered off
- EVGA have persistent memory
- liquidctl can control them from macOS

But you've already written it, so have no real issue.

> It is impossible to control AIO kits from the motherboard. They are not designed this way. The three pin connector for the CPU header is only there to report the Pump and Radiator fan speeds. It is a one-way communication. It does not respond to Voltage or PWM controls.

Older kits(not relying on USB) have the option to control pump speed in the BIOS, was how I controlled my old AIO(Some thermaltake 120mm). I'd just say "the majority of AIOs can't be controlled"

Besides that last point regarding pump control, would be happy to merge.

**Comment by icedterminal** [comment]
The goal was to prevent reader questioning or confusion. So I made sure to include detail. However, I did trim down while trying to keep some detailed information.

File: Cooler.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 
-It is impossible to control AIO kits from the motherboard. They are not designed this way. The three pin connector for the CPU header is only there to report the Pump and Radiator fan speeds. It is a one-way communication. It does not respond to Voltage or PWM controls. The only way to control these variables is through the USB header with software.
+There are Python based utilities for controlling your kit on GitHub. Though this is usually unnecessary. Setting it up once in Windows will be enough until you turn off the PSU. Python based [liquidctl](https://github.com/jonasmalacofilho/liquidctl) is developed for macOS and works for a few models from NZXT, EVGA, and Corsair.
```
Comment (dhinakg) [inline Cooler.md:12]:
```suggestion
There are Python based utilities for controlling your kit on GitHub, though this is usually unnecessary, as setting it up once in Windows will be enough until you turn off the PSU. Python based [liquidctl](https://github.com/jonasmalacofilho/liquidctl) is developed for macOS and works for a few models from NZXT, EVGA, and Corsair.
```

Comment (icedterminal) [reply Cooler.md]: Thanks

**Comment by khronokernel** [comment]
Sorry it took this long to merge, got caught up with other things. Thanks so much for the PR!

**Pull request merged**